<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>412</height>
   </rect>
  </property>
  
  <property name="windowTitle">
   <string>RCAT</string>
  </property>
  
  <property name="windowIcon">
   <iconset>
    <normaloff>logo2.PNG</normaloff>logo2.PNG</iconset>
  </property>
  
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>230</height>
       </size>
      </property>
      
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="tw">
         <property name="minimumSize">
          <size>
           <width>370</width>
           <height>100</height>
          </size>
         </property>
         
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>File</string>
          </property>
          
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          
          <property name="background">
           <color>
            <red>4</red>
            <green>60</green>
            <blue>145</blue>
           </color>
          </property>
         </column>
         
         <column>
          <property name="text">
           <string>Status</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
       
       <item row="0" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       
       <item row="0" column="1">
        <widget class="QPushButton" name="Delete">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       
       <item row="0" column="2">
        <widget class="QPushButton" name="all">
         <property name="toolTip">
          <string>Recommend Clear Panel before re-Run</string>
         </property>
         <property name="whatsThis">
          <string comment="fdfd">what is this</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
    
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(255, 0, 0); gridline-color: rgb(255, 0, 0); border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="title">
          <string> Parameters: </string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
          
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_1">
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Time points(in 24hr):</string>
            </property>
           </widget>
          </item>
          
          <item row="0" column="1">
           <widget class="QLineEdit" name="point">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>points</string>
            </property>
           </widget>
          </item>
          
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>Analytic range:</string>
            </property>
           </widget>
          </item>
          
          <item row="1" column="1">
           <widget class="QLineEdit" name="startPoint">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          
          <item row="1" column="2">
           <widget class="QLineEdit" name="endPoint">
            <property name="text">
             <string>48</string>
            </property>
           </widget>
          </item>
          
          <item row="1" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>hr</string>
            </property>
           </widget>
          </item>
          
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Period of interest:</string>
            </property>
           </widget>
          </item>
          
          <item row="3" column="1">
           <widget class="QLineEdit" name="minPeriod">
            <property name="text">
             <string>16</string>
            </property>
           </widget>
          </item>
          
            <item row="3" column="2">
           <widget class="QLineEdit" name="maxPeriod">
            <property name="text">
             <string>32</string>
            </property>
           </widget>
          </item>
          
          <item row="3" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>hr</string>
            </property>
           </widget>
          </item>
          
          <item row="4" column="1" colspan="2">
           <widget class="QRadioButton" name="pic">
            <property name="text">
             <string>Output Img</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
       
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>156</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       
       <item>
        <widget class="QPushButton" name="run">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;One-key Execute&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>RUN</string>
         </property>
        </widget>
       </item>
       <item>
       
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>203</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 213, 204);</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="importFile"/>
    <addaction name="resultFile"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   
   <widget class="QMenu" name="menuHelp">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 213, 204);</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(192, 192, 192); background-color: rgb(192, 192, 192);</string>
   </property>
  </widget>
  
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  
  <action name="importFile">
   <property name="icon">
    <iconset theme="input">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  
  <action name="resultFile">
   <property name="text">
    <string>ResultFile</string>
   </property>
  </action>
  
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  
  <action name="resultSave">
   <property name="text">
    <string>ResultSave AS</string>
   </property>
  </action>
  
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
